### PR TITLE
.Net: Empty projects for the new AzureOpenAI connector

### PR DESCRIPTION
**Pull request by SergeyMenshykh**
Empty Connectors.AzureOpenAI and Connectors.AzureOpenAI.UnitTests projects as a first step to start building AzureOpenAI conector based on new AzureOpenAI SDK.

### CODE REVIEW

File: dotnet/SK-dotnet.sln [hunk 1]
```diff
@@ -803,6 +807,18 @@ Global
 		{FDEB4884-89B9-4656-80A0-57C7464490F7}.Publish|Any CPU.Build.0 = Debug|Any CPU
 		{FDEB4884-89B9-4656-80A0-57C7464490F7}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{FDEB4884-89B9-4656-80A0-57C7464490F7}.Release|Any CPU.Build.0 = Release|Any CPU
+		{6744272E-8326-48CE-9A3F-6BE227A5E777}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{6744272E-8326-48CE-9A3F-6BE227A5E777}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{6744272E-8326-48CE-9A3F-6BE227A5E777}.Publish|Any CPU.ActiveCfg = Debug|Any CPU
+		{6744272E-8326-48CE-9A3F-6BE227A5E777}.Publish|Any CPU.Build.0 = Debug|Any CPU
```
Comment (dmytrostruk) [inline dotnet/SK-dotnet.sln:813]: Little issue, but it causes big problems in our release pipeline :) It's not important for test projects, but it's important for projects that we pack and release.
```suggestion
		{6744272E-8326-48CE-9A3F-6BE227A5E777}.Publish|Any CPU.ActiveCfg = Publish|Any CPU
		{6744272E-8326-48CE-9A3F-6BE227A5E777}.Publish|Any CPU.Build.0 = Publish|Any CPU
```

File: dotnet/src/Connectors/Connectors.AzureOpenAI/Connectors.AzureOpenAI.csproj [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Connectors.AzureOpenAI</AssemblyName>
+    <RootNamespace>$(AssemblyName)</RootNamespace>
+    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <NoWarn>$(NoWarn);NU5104;SKEXP0001,SKEXP0010</NoWarn>
+    <EnablePackageValidation>false</EnablePackageValidation>
+  </PropertyGroup>
```
Comment (dmytrostruk) [inline dotnet/src/Connectors/Connectors.AzureOpenAI/Connectors.AzureOpenAI.csproj:11]: I don't remember the decision about this if any was made, but do we need version suffix for this package like `<VersionSuffix>alpha</VersionSuffix>` or `<VersionSuffix>preview</VersionSuffix>`?

Comment (SergeyMenshykh) [reply dotnet/src/Connectors/Connectors.AzureOpenAI/Connectors.AzureOpenAI.csproj:11]: It seems we've discussed it and decided to go straight to the released one. cc: @markwallace-microsoft

Comment (markwallace-microsoft) [reply dotnet/src/Connectors/Connectors.AzureOpenAI/Connectors.AzureOpenAI.csproj]: This package will replace code that is already at 1.x level so we need to be back at that level when we merge this back to main. If we decide to release from this branch we may add a suffix but we can add that if needed later.

**Pull request merged**